src/main/java/io/group3/backend/model/Greeting.java

```diff
@@ -0,0 +1,21 @@
+package io.group3.backend.model;
+
+public class Greeting {
+
+    private int id;
+    private String msg;
+
+    public Greeting(int id, String msg) {
+        this.id = id;
+        this.msg = msg;
+    }
+
+    public String getMsg() {
+        return msg;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+}
```

Comment:
I'm not fan of get set. there're alternative way to us it as POJO.

Reply:
Thank you for the suggestion. I've already updated our model file by remove getter/setter and changed access modified of properties to Public.